runtime/testing/e2e/a11y.ts

```diff
@@ -0,0 +1,46 @@
+/*jslint node: true */
+'use strict';
+
+const AxeBuilder = require('axe-webdriverjs');
+import { browser } from 'protractor';
+
+export class A11yTest {
```

Comment:
Sounds good. Going to go with SkyA11y. I intentionally didn't include "axe" anywhere as I thought this Sky class could down the road include other a11y testing libraries within it and teams wouldn't need to know those details and just have this one A11y call.